Assets/Scripts/UIAndMenus/EscMenuController.cs

```diff
@@ -47,6 +63,6 @@ public void CloseSettings()
 
     public void Quit()
```

Comment:
may want to change this to something other than Quit (like ReturnToMainMenu() maybe? unless you can come up with something better) to differentiate it from quitting the application
(probably not strictly needed, though--more of a stylistic choice)